lib/src/util/withClass.tsx

```diff
@@ -13,7 +13,20 @@ export default function withClass<E extends keyof JSX.IntrinsicElements>(
 ): FunctionComponent<JSX.IntrinsicElements[E]> {
   return function WithClassWrapper({ className, ...otherProps }: JSX.IntrinsicElements[E]) {
     const classes = classnames(withClassName, className);
-
     return React.createElement(El, { className: classes, ...otherProps});
   };
 }
+
+export function withClassAndForwardedRef<E extends keyof JSX.IntrinsicElements, R extends HTMLElement = HTMLElement>(
```

Comment:
Let me know what your thoughts are on this. I'm still trying to figure out how to derive specific subset of HTMLElement interface JSX.IntrinsicElements key, in the meantime for testing I just used HTMLElement